### PR TITLE
Create: Change copy for payouts

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2510** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2510" target="_blank">Graphite</a> 
    * **PR #2511** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2511" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2511?utm_source=stack-comment).

File: src/components/Create/Create.tsx [hunk 1]
```diff
@@ -120,8 +120,9 @@ export function Create() {
                   }}
                 >
                   <Trans>
-                    Choose which addresses to pay and how to split the total
-                    payout amount each funding cycle. How do I decide?
+                    Start by selecting how you would like to distribute your
+                    payouts and then choose which Ethereum wallets you would
+                    like to pay each funding cycle.
```
Comment (tomquirk) [inline src/components/Create/Create.tsx:125]:
```suggestion
                    Select how you would like to distribute
                    payouts. Then, choose which Ethereum wallets to pay each funding cycle.
```

**Review by tomquirk — CHANGES_REQUESTED** [review]
See inline comments

**Review by tomquirk — APPROVED** [review]

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2511?utm_source=stack-comment).


Job Reference: XJjT605R1Sdi2CUgZUbX

**Pull request merged**